tox.ini

```diff
@@ -12,7 +12,7 @@ commands =
     pytest --cov
 
 [flake8]
-exclude = .tox
+exclude = .tox,.git,__pycache__,build,sqllineagejs
```

Comment:
Any particular reasons we want to add this?

Reply:
I found that just running `flake8` tries to entire directories so it takes over minutes for flake8 because it scans `build` and `.venv`, etc. Thus I introduces some exclude to eliminate on tox like this:

```
py310 run-test: commands[1] | flake8
./build/lib/sqllineage/core/handlers/cte.py:1:1: I101 Imported names are in the wrong order. Should be Function, Identifier, IdentifierList, Parenthesis, Token
from sqlparse.sql import Function, Identifier, IdentifierList, Token, Parenthesis
^
ERROR: InvocationError for command /Users/ariga/src/sqllineage/.tox/py310/bin/flake8 (exited with code 1)
________________________________________________________________________ summary _________________________________________________________________________
ERROR:   py310: commands failed
```

Or, it'd be better to set explicit targets like:

```
flake8 sqllineage tests docs setup.py
```

Reply:
I see. Mostly I think it's .venv, which includes tons of Python package code. Mine wouldn't take minutes. But sure, we can have the explicit exclusion here.